tests/integration/common_tasks/common_tasks/test_tasks/fib.py

```diff
@@ -0,0 +1,19 @@
+from girder_worker.utils import girder_job
+from girder_worker.app import app
+
+
+@girder_job(title='Fibonacci Job')
+@app.task
+def fibonacci(n, **kwargs):
+    if n == 1 or n == 2:
+        return 1
+    return fibonacci(n-1) + fibonacci(n-2)
+
+
+@app.task(bind=True)
+def fib_seq(task, n, **kwargs):
```

Comment:
This is a silly point, but I've noticed we use `task` when all of the Celery documentation I've seen uses `self` when defining bound tasks, should we start transitioning towards that moving forward?